### DISCUSSION TITLE
Any news on 1.0 release?

### DISCUSSION THREAD

**Discussion by Takore2002** [toverainc/willow#239] [General]
I've been seeing all those PRE release alerts and have been getting hyped for the 1.0 release. Are there any updates on the timeline for when that will be rolling out? The last I remember hearing was it should be the end of July.

**Comment by stintel** [comment]
We have finally merged the feature/was branch and tagged [0.1.0-rc.1](https://github.com/toverainc/willow/releases/tag/0.1.0-rc.1). See [Quick Start Guide](http://heywillow.io/quick-start-guide/) how to use it.
